interfaces/matlab/toolbox/importPhase.m

```diff
@@ -13,6 +14,7 @@
 %     Instance of class :mat:func:`Solution`
 %
 
+warning('This function is deprecated and will be removed after Cantera 2.3.');
```

Comment:
I think it should be `warning('This function is deprecated and will be removed after Cantera 2.3. Use Solution instead.')`